lib/internal/bootstrap/web/exposed-window.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const {
+  globalThis,
+} = primordials;
+
+const {
+  defineOperation,
+  defineLazyProperties,
+  defineReplaceableLazyAttribute,
+  exposeLazyInterfaces,
+  exposeNamespace,
+} = require('internal/util');
+const config = internalBinding('config');
+
+// https://console.spec.whatwg.org/#console-namespace
```

Comment:
This is actually `[Exposed=*]`, but I guess we don't want to install it yet because this needs too many risky internals? It'd be good to explain a bit in the comment.

Reply:
Thanks, somehow I overlooked this one. Fixed.